libs/common-services/src/main/java/uk/gov/di/ipv/core/library/persistence/item/IpvSessionItem.java

```diff
@@ -41,7 +39,7 @@ public class IpvSessionItem implements DynamodbItem {
     // Only for passing the featureSet to the external API lambdas at the end of the user journey.
     // Not for general use.
     private String featureSet;
-    private List<String> vcReceivedThisSession;
+    private boolean inheritedIdentityReceivedThisSession;
```

Comment:
I think this is fine for now - we might need to rethink this entirely once we introduce EVCS though, see notes on https://govukverify.atlassian.net/browse/PYIC-6083

Reply:
I figured that as we're not live with inherited identities this should be fine. The reads and writes to this are with behind a flag, or only read if the user has an operation profile.